vipps-checkout-api.md

```diff
@@ -94,6 +94,16 @@ These options may be combined with shipping if it fits your scenario. For exampl
 
 Vipps Checkout supports easy fetching of user info with the built-in Vipps Login integration. With a functionality called "Remember Me", the user is can opt in to having this information being persisted across different Vipps Checkout sessions on the same machine.
 
+### Prefill customer data
+
+We believe that simplicity is key to providing the best user experience. With `prefillCustomer`, you can easily send in any information you have about your customer, making filling out forms in the Checkout a breeze.
+All fields in the `prefillCustomer` optional. We will prefill the form with whatever information you send and let your customers fill in the rest if some information is missing.
```

Comment:
All fields in the 'prefillCustomer' **are** optional

Reply:
And 'if any information is missing.'

Reply:
Good catch, will fix!